src/containers/create-new/index.tsx

```diff
@@ -72,6 +76,8 @@ const CreateNewArea = ({ className, space }: Props) => {
 
   const handleTagsChange = (tagProps: string[]) => {
     setTags(tagProps);
+    if (tags.length + 1 === 8) setTagErrMessage('You can not set more than 8 tags');
```

Comment:
This messages appear when I have 8 tags selected, but I don't think so is the correct approach because you are showing a unnecesary message.

I think it’s better if the message appear just when I want to add the 9 tag.

Reply:
Yep, that's what the function does